extendr-polars/Cargo.toml

```diff
@@ -10,8 +10,8 @@ description = "extendr bindings to polars"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-polars = { version = "*", default_features = false, features = ["dtype-struct"]}
-polars-core = { version = "*", default_features = false }
```

Comment:
These changes are needed?

Reply:
cargo warned me that this parameter is not available for polars & polars_core. To be honest: I am unsure about the implications of these changes and just wanted to remove compiler warnings along the way.

Reply:
I don't think this change is ideal here because the default functionality includes extra stuff.